docs/sources/mimir/reference-configuration-parameters/index.md

```diff
@@ -2826,6 +2831,8 @@ The `limits` block configures default and per-tenant limits imposed by component
 # Rules based on which the Distributor decides whether a metric should be
 # forwarded to an alternative remote_write API endpoint.
 [forwarding_rules: <map of string to validation.ForwardingRule> | default = ]
+
+ephemeral_series_matchers:
```

Comment:
The documentation is empty here. Can you fix it, please?

Reply:
Looking into why this doesn't pick up the description from the CLI flag definition...

Reply:
Fixed here, I needed to update the handling of custom types in the doc-generator to make it work: https://github.com/grafana/mimir/pull/3897/commits/01b56858d43a97104dd033f6de2c0d7547944f39